test/e2e/tls_test.go

```diff
@@ -195,8 +195,8 @@ var _ = Describe("TLS activation - operator", Ordered, func() {
 		By("forwarding the HTTPS port to a local port")
 		var cleanUp func()
 		forwardHost, cleanUp, err = client.ForwardPort(operatorServiceName, OLSNameSpace, operatorServicePort)
-		Expect(err).NotTo(HaveOccurred())
 		cleanUpFuncs = append(cleanUpFuncs, cleanUp)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
I think this wont help, either.

Reply:
if we want to kill the oc process, return a cleanup function when error happends and keep this change.

Reply:
Yes, you are right. I forgot to remove this one. Thank you.